### DISCUSSION TITLE
New Page Ideas :)

### DISCUSSION THREAD

**Discussion by nicedevil007** [tteck/Proxmox#2969] [Ideas]
Hey @tteck,

it's been a while since my last PR and I saw your new page now. Would it be possible to make the "scripts" page the default one for your new URL? Your cool animation could be moved directly under the "new scripts" part. So we can see if anything new usefull is released and you still got your entry part located anywhere?

Other question I have: Would it be possible to sort the scripts inside the categories by alphabetical order?

![image](https://github.com/tteck/Proxmox/assets/17103076/01c10ade-ec3b-4b58-9d99-0e596087f84a)

**Comment by tteck** [comment]
> Would it be possible to make the "scripts" page the default one for your new URL?

Just change your bookmark to https://helper-scripts.com/scripts

> Would it be possible to sort the scripts inside the categories by alphabetical order?

Yes, that's possible

**Reply by tteck** [reply]
@nicedevil007 The scripts inside the categories are now in alphabetical order.

**Reply by nicedevil007** [reply]
Thank you! Just clicked through and looking good! Maybe just one missed category here:

![image](https://github.com/tteck/Proxmox/assets/17103076/97187f94-b9a1-44c1-b439-6ff04eef43f2)

**Reply by tteck** [reply]
Yeah "Proxmox VE Tools" and "Home Assistant" stay the same.

**Comment by MickLesk** [comment]
@tteck 

1.) Wouldn't it be good to have an overview page of “ideas”? This would allow Github users to vote or contribute new ideas. (Similar to a poll option in forum software)? 

I think the post is very confusing now, with lots of double, triple... contributions to the same request. 
https://github.com/tteck/Proxmox/discussions/50


2.) The Contribute button redirects you to Github, with a short note. I had the feeling that I didn't need this contribution. 
=> Maybe add it directly to the page?
=> As a hint: Maybe provide a “blank script” for each type, with a few “hints” (e.g. the font editor for the headings (ASCII)) 

3.) I would also integrate guides directly on the page, similar to each script in alphabetical order with a heading
e.g:
HomeAssistant
-> Guide 1
-> Guide 2
-> ... 

4.) You could also outsource changelogs to the page, simply with spoilers. (Github additionally)

Latest entry -> Without spoilers
after that:
[SPOILER_WITH_DATE] 
Contents
[SPOILER_END]

**Reply by nicedevil007** [reply]
I like some of your ideas, but keep in mind tteck is doing all of this in his free time, and I guess the new page took quite some time to make it like it is right now. I can absolutely understand that sometimes (doesn't matter if it is a good idea or not), one shouldn't invest more time to fulfill everybody's wishes.

Keep it some time maybe in the future he is more open to do something (maybe not all, or even nothing) like this :)

**Reply by tteck** [reply]
To be honest, I didn't like any of those ideas 🤷🏻‍♂️. Oh, and this is a two man team on this website project, @BramSuurdje & @tteck, with Bram doing most of the heavy lifting.

Thank you Bram!
